tree/dataframe/test/dataframe_interface.cxx

```diff
@@ -36,7 +36,7 @@ TEST(RDataFrameInterface, CreateFromContainer)
 
 TEST(RDataFrameInterface, CreateFromInitList)
 {
-   RDataFrame tdf("t", {"f1", "f2"});
+   RDataFrame tdf("t", {{"f1"}, {"f2"}});
```

Comment:
This might not be what we want. @eguiraud ? (I.e. yes it might fix the build but we might want to change the API.)

Reply:
At least according to MSVC: indeed.

I don't think GCC or clang complained about that (and as a consequence they _will_ likely complain about this change).